docs/design/flow-monitoring/isl-cache-update.puml

```diff
@@ -1,16 +1,24 @@
 @startuml
```

Comment:
Don't see "rendered" variant of this file.

Reply:
Added.